codespell_lib/data/dictionary.txt

```diff
@@ -17999,6 +18008,11 @@ hardward->hardware
 hardwdare->hardware
 hardwirted->hardwired
 harge->charge
+harrang->harangue
+harranged->harangued
+harranger->haranguer
+harranges->harangues
```

Comment:
In a single case in GitHub, the proper fix was actually `arranges`.

.

Reply:
> In a single case in GitHub, the proper fix was actually `arranges`.

Added.